react/features/follow-me/subscriber.js

```diff
@@ -51,6 +52,16 @@ StateListenerRegistry.register(
     /* selector */ state => state['features/filmstrip'].visible,
     /* listener */ _sendFollowMeCommand);
 
+/**
+ * Subscribes to changes to the stage filmstrip participants.
+ */
+StateListenerRegistry.register(
+    /* selector */ state => state['features/filmstrip'].activeParticipants,
+    /* listener */ _sendFollowMeCommand,
```

Comment:
What is the send rate of the messages? How often will this happen?

Reply:
On dominant speaker change and on pin participant

Reply:
How often is that happening is what I ask? ... In 500 participants call sending this too often is very costly for the server.

Reply:
The way it will work is: moderator enables follow me, and then pins multiple participants. The list of pinned participants is sent to everyone. How often will this happen? No idea

Reply:
How often the list will be updated through the presence? Or once the moderator pins some people that list is sent and never updated?